core/templates/dev/head/pages/library/Library.js

```diff
@@ -16,16 +16,19 @@
  * @fileoverview Data and controllers for the Oppia contributors' library page.
  */
 
-// NOTE TO DEVELOPERS: The constants defined below in LIBRARY_PAGE_MODES should
-// be same as the LIBRARY_PAGE_MODE constants defined in feconf.py. For example
-// LIBRARY_PAGE_MODES.GROUP should have the same value as
-// LIBRARY_PAGE_MODE_GROUP in feconf.py.
 oppia.constant('LIBRARY_PAGE_MODES', {
   GROUP: 'group',
   INDEX: 'index',
   SEARCH: 'search'
 });
 
+oppia.constant('LIBRARY_PATHS_AND_MODES', {
```

Comment:
I'm not sure whether it's a good name for this dict. maybe we can have `LIBRARY_PATH_TO_MODE`. (not sure!)

@seanlip, any suggestion?

Reply:
LIBRARY_PATHS_TO_MODES

Reply:
Done.